pkg/k8sutil/k8sutil.go

```diff
@@ -154,6 +155,13 @@ func GetGVKsFromAddToScheme(addToSchemeFunc func(*runtime.Scheme) error) ([]sche
 		if !isKubeMetaKind(gvk.Kind) {
 			ownGVKs = append(ownGVKs, gvk)
 		}
+
+		for _, customKind := range customMetaKinds {
+
+			if customKind == gvk.Kind {
+				ownGVKs = append(ownGVKs, gvk)
+			}
+		}
 	}
 
```

Comment:
Also, it is missing:
- Update the doc with this information [here](https://github.com/operator-framework/operator-sdk/blob/master/doc/user-guide.md#default-metrics-exported-with-3rd-party-resource). See that the same change need to be replicated to the dir .website [here](https://github.com/operator-framework/operator-sdk/blob/master/website/content/en/docs/golang/quickstart.md#default-metrics-exported-with-3rd-party-resource) 
-  Add CHANGELOG](https://github.com/operator-framework/operator-sdk/blob/master/CHANGELOG.md#added) entry. 
- Improve the comment of this method in order to make clear that it is possible to pass a list of GKVs that would be ignored/remove as well.

Reply:
Thank you for providing the details about the missing points. I have made the recommended changes.